GPT/beta-commands/beta-cursorless.talon

```diff
@@ -14,3 +14,22 @@ app: vscode
     text_list = user.cursorless_get_text_list(cursorless_target)
     default_destination = user.cursorless_create_destination(cursorless_target)
     user.gpt_dynamic_request_cursorless(user.text, text_list, cursorless_destination or default_destination)
+
+model blend <user.cursorless_target> to <user.cursorless_target>$:
+    target_text = user.cursorless_get_text_list(cursorless_target_1)
+    destination_text = user.cursorless_get_text(cursorless_target_2)
+    default_destination = user.cursorless_create_destination(cursorless_target_2)
+    result = user.model_blend(target_text, destination_text)
+    user.cursorless_insert(default_destination, result)
+
+model blend to <user.cursorless_target>$:
+    target_text = edit.selected_text()
+    destination_text = user.cursorless_get_text(cursorless_target)
+    default_destination = user.cursorless_create_destination(cursorless_target)
+    result = user.model_blend(target_text, destination_text)
+    user.cursorless_insert(default_destination, result)
+
+model blend <user.cursorless_target>$:
+    target_text = user.cursorless_get_text_list(cursorless_target)
```

Comment:
```suggestion
    target_text = user.cursorless_get_text(cursorless_target)
```

Reply:
I was on the fence about using the singular version because I've found that the list call seems to work just fine here but also allows you to specify multiple targets (where the singular api call throws an error). Is this a better choice?

Reply:
That's fine you should just update the type hint for the first arg on `model_blend` to indicate it's a list of strings, not a string. Tbh I'm surprised talon doesn't complain about the incorrect runtime type there

Reply:
Is there a way to create a list of string in talon script? For the clipboard usage I was struggling with how to do that.

Reply:
I think I'll make another function for the list use case. I can provide more context to the LLM that way anyhow.